src/entities/removeLiquidity/removeLiquidity.ts

```diff
@@ -22,12 +23,11 @@ export class RemoveLiquidity {
         const { customRemoveLiquidityTypes } = config || {};
         this.removeLiquidityTypes = {
             //GYRO2, GYRO3, GYROE only support Remove Liquidity Proportional(1 - EXACT_BPT_IN_FOR_TOKENS_OUT)
-            GYRO2: new RemoveLiquidityWeighted(),
-            GYRO3: new RemoveLiquidityWeighted(),
-            GYROE: new RemoveLiquidityWeighted(),
-            WEIGHTED: new RemoveLiquidityWeighted(),
-            // PHANTOM_STABLE === ComposableStables in API
-            PHANTOM_STABLE: new RemoveLiquidityComposableStable(),
+            [PoolType.Gyro2]: new RemoveLiquidityWeighted(),
+            [PoolType.Gyro3]: new RemoveLiquidityWeighted(),
+            [PoolType.GyroE]: new RemoveLiquidityWeighted(),
+            [PoolType.Weighted]: new RemoveLiquidityWeighted(),
+            [PoolType.ComposableStable]: new RemoveLiquidityComposableStable(),
```

Comment:
Nice solution! We use poolTypes while letting the user provide custom types as string 👏

Reply:
Thanks! 😊